soaks/tests/file_to_blackhole/README.md

```diff
@@ -0,0 +1,9 @@
+# File -> Blackhole
```

Comment:
Will this be sourcing a single file at a time or multiple files in parallel - I imagine performance will be different depending on the number of files Vector reads from.

Reply:
Multiple files. I wrote the config to write four at once.